src/cl_wrapper/cl_interceptor.c

```diff
@@ -1699,63 +1767,85 @@ clSVMAlloc(cl_context           context,
     void *ret = NULL;
     if ( SVMAlloc )
     {
+        size_t size_aug = size;
+
         initialize_logging();
         // Workaround for a strange bug in the AMD OpenCL runtime.
         // If we start allocating small SVM buffers, some memory gets corrupted
         // This is exemplified by fir_cl20, which will segfault in the OpenCL
         // runtime iff you use some of its smaller buffers.
         // This workaround likely does not fix the problem, and may breka some
         // of your apps that really do require small SVM buffers. Sorry..
+        /*
```

Comment:
We shouldn't just keep random commented code around.
This "workaround" should be done if we're on the AMD Catalyst platform -- and maybe the amdgpu-pro and/or ROCm platforms. We should test to see which platforms this is required on.

As I mentioned during the last attempt at pushing this, I consider this a NACK on this patch until this is fixed. If Catalyst still needs this workaround, then commenting this out breaks Catalyst. If Catalyst doesn't need this patch, let's delete the whole thing.